guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<ul><li>dataType is mentioned as integer and default value is a string here</li><li>Quantity can be quantity to maintain consistency</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Changed it to lower case</p>